test/dune

```diff
@@ -268,3 +268,12 @@
 (alias
  (name runtest)
  (action (diff dune_rules.inc dune_rules.inc.gen)))
+
+(alias
```

Comment:
Is this generated or added by hand?

Reply:
By hand for now. See https://github.com/realworldocaml/mdx/pull/162